web/views/page_view.ex

```diff
@@ -1,11 +1,16 @@
 defmodule DevWizard.PageView do
   use DevWizard.Web, :view
+  use Timex
 
-  def review_state(issue) do
-    case issue.review_state do
-      :not_signed_off -> "Needs Review!"
-      :signed_off     -> "Signed Off"
-      _               -> "?"
+  def days_since(date) do
```

Comment:
This helper is really nice. I like having it and I think displaying this information as "some amount of time ago" is the way to go. I'd like to think longer term if days and or hours is a more interesting measurement, but I think this is a great place to start. 👍